js/stores/appStore.js

```diff
@@ -253,7 +257,7 @@ function windowDefaults () {
     show: false,
     width: appState.get('defaultWindowWidth'),
     height: appState.get('defaultWindowHeight'),
-    minWidth: 480,
+    minWidth: isWindows ? 640 : 480, // Windows has caption buttons (min/max/close) which take extra space
```

Comment:
change backed out- solution was to (on Windows only) reduce the size of elements so that the window can go as skinny as 480 and still look correct